<launch>
	<node pkg="can_transmit" type="can_transmit_node" name="can_transmit_test" output="screen">
        <param name="cmd_topic" type="string" value="/cmd_vel"/>
		<param name="rune_cmd_topic" type="string" value="/rune_cmd"/>
    </node>

	<node pkg="can_receive" type="can_receive_node" name="can_receive_1" output="screen">
		<param name="can_device" value="can1"/>
		<param name="receiver_topic" type="string" value="/can1_raw"/>
	</node>

	<node pkg="socketcan_bridge" type="socketcan_bridge_node" name="double_comm" output="screen">
		<param name="can_device" value="can1"/>
	</node>
</launch>
